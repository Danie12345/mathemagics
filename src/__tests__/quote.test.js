import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';

const Quote = require('../components/quote/Quote').default;

test('loads the quotes with everything', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
