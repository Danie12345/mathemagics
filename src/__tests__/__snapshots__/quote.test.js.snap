// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loads the quotes with everything 1`] = `
<div
  className="section quoteContainer"
>
  <p
    className="quote"
  >
    <span
      className="quote-content"
    >
      An equation means nothing to me unless it expresses a thought of God.
    </span>
    <small
      className="quote-mathematician"
    >
      - Srinivasa Ramanujan
    </small>
  </p>
</div>
`;
